csiplugin_k8s.yaml

```diff
@@ -327,6 +327,66 @@ spec:
         #    type: Directory
 
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: csi-etcd-svc
+  namespace: csi-datenlord
+spec:
+  selector:
+    app: csi-etcd
+  ports:
+    - protocol: TCP
+      port: 2379
+      targetPort: 2379
+
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
```

Comment:
etcd StatefulSet needs to mount hostPath volume to store etcd data

Reply:
Done